frontend/src/features/whats-new/utils/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { isAfter } from 'date-fns'
+
+import { UserDto } from '~shared/types/user'
+
+import { FeatureUpdateList } from '../FeatureUpdateList'
+
+export interface shouldShowLatestFeatureUpdateNotificationProps {
+  user: UserDto | undefined
+  isLoading: boolean
+}
+
+export const shouldShowLatestFeatureUpdateNotification = ({
+  user,
+  isLoading,
+}: shouldShowLatestFeatureUpdateNotificationProps): boolean => {
+  if (isLoading) {
+    return false
+  }
+  const sortedFeatureUpdateList = FeatureUpdateList.sort(
+    (featureUpdateA, featureUpdateB) =>
+      Date.parse(featureUpdateB.date) - Date.parse(featureUpdateA.date),
+  )
```

Comment:
suspect we don't need to sort. We should always just check the first feature list's date and it should be fine.